playbook.md

```diff
@@ -2247,51 +2247,9 @@ If there is a book relevant to your work that you would like to read, make a req
 
 dxw will pay for the equipment needed to enable you to work comfortably and safely. dxw also offers a budget of £25 towards a work backpack. Use the `dxw-purchasing` Slack channel to request any equipment you need, or talk to anyone in business operations if you are unsure if a purchase will be covered.
 
-## Sharing
+[Sharing](sections/sharing.md ':include')
```

Comment:
I'm struggling to find any information about this `':include'` or what it does. Could you point me in the right direction?